### PR TITLE
Extend tss--sync-server with custom source

### PR DESCRIPTION
**Pull request by carltonf**
See added inline doc of tss--sync-server for details.

Currently I'm implementing a company-backend for emacs-tss, this patch makes completions more responsive and paves way to have more documentation details in the company-doc.

### CODE REVIEW

File: tss.el [hunk 1]
```diff
@@ -533,7 +533,20 @@
                (tss--trace "Got response from server.")
                tss--server-response))))))
 
-(defun* tss--sync-server (&key waitsec path buff)
+(defun* tss--sync-server (&key waitsec buff path
+                               source linecount)
+  "Sync current source with tss service. Default to 15 waiti,
```
Comment (meteficha) [inline tss.el]: What is a "waiti"?  Is it a typo?  (I've never programmed elisp.)

Comment (carltonf) [inline tss.el]: Abbrebriate for "wait Interval" I believe... I follow the convention is the code. Refer to the code below there is waiti and maxwaiti, and they are used as counter in the while loop further down. So I think actually "waitsec" is the misleading name here (it's not used as secs ...).
(PS: just naming issue, not elisp specific)

Comment (aki2o) [inline tss.el]: I'm sorry to reply very late to your contacts and confuse you by my laziness...
It means "wait index" as loop counter like `(loop for i from 1 to ...)`.

**Comment by aki2o** [comment]
I'm sorry I'm so late...

**Pull request merged**